src/java.base/share/classes/java/lang/invoke/VarHandles.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2014, 2018, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2014, 2022, Oracle and/or its affiliates. All rights reserved.
```

Comment:
This file has a copyright year change, but no actual changes.

Reply:
Good catch, thanks.